pom.xml

```diff
@@ -164,9 +164,11 @@
         <version>3.1.0</version>
       </dependency>
       <dependency>
-        <groupId>junit</groupId>
-        <artifactId>junit</artifactId>
-        <version>4.13.2</version>
```

Comment:
Any particular reason this was removed from dependency management?

junit-bom does not include a `junit` version